string robot_name
uint16 id
float32[] pose
float32[] local_free_space_rect